### PR TITLE
Update cloud-provider-aws docs

### PR DESCRIPTION
**Pull request by mcrute**
This is the start of the cloud provider docs for the 1.13 release.

/milestone v1.13
/kind feature
/sig aws
/sig cloud-provider

### CODE REVIEW

**Comment by darwin67** [comment]
`1.13.2` is already out and `1.14.0` is on it's way. why is this still on hold?

**Comment by d-nishi** [comment]
@mcrute can you please do the needful per comments.

This has been pending for a while now.

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Amazon Cloud Controller Manager
```
Comment (darwin67) [inline docs/README.md]: @mcrute why did you close this?
could you at least put this file in so it doesn't mislead people that this is currently usable?

**Pull request closed without merging**